sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client.py

```diff
@@ -27,12 +27,68 @@
 )
 import pytest
 import datetime
+import os
+import logging
 
-class AppConfigurationClientTest(AzureAppConfigurationClientTestBase):
+class AppConfigurationClientTest(AzureMgmtTestCase):
     def __init__(self, method_name):
-        super(AppConfigurationClientTest, self).__init__(
-            method_name, AzureAppConfigurationClient
+        super(AppConfigurationClientTest, self).__init__(method_name)
+        self.vcr.match_on = ["path", "method", "query"]
+        if self.is_playback():
+            connection_str = "Endpoint=https://fake_app_config.azconfig-test.io;Id=0-l4-s0:h5htBaY5Z1LwFz50bIQv;Secret=bgyvBgwsQIw0s8myrqJJI3nLrj81M/kzSgSuP4BBoVg="
+        else:
+            connection_str = os.getenv('APP_CONFIG_CONNECTION')
+        self.app_config_client = AzureAppConfigurationClient.from_connection_string(connection_str)
+
+    def setUp(self):
+        super(AppConfigurationClientTest, self).setUp()
+        self.test_config_setting = self._add_for_test(
+            ConfigurationSetting(
+                key=KEY,
+                label=LABEL,
+                value=TEST_VALUE,
+                content_type=TEST_CONTENT_TYPE,
+                tags={"tag1": "tag1", "tag2": "tag2"},
+            )
+        )
+        self.test_config_setting_no_label = self._add_for_test(
+            ConfigurationSetting(
+                key=KEY,
+                label=None,
+                value=TEST_VALUE,
+                content_type=TEST_CONTENT_TYPE,
+                tags={"tag1": "tag1", "tag2": "tag2"},
+            )
         )
+        self.to_delete = [self.test_config_setting, self.test_config_setting_no_label]
+
+    def tearDown(self):
+        super(AppConfigurationClientTest, self).tearDown()
+        for item in self.to_delete:
```

Comment:
Can't you have key specific per tests and delete the entire thing at the end?

Reply:
Will be done in https://github.com/Azure/azure-sdk-for-python/issues/9062